pysat/instruments/nasa_cdaweb_methods.py

```diff
@@ -260,27 +262,61 @@ def download(supported_tags, date_array, tag, sat_id,
         saved_local_fname = os.path.join(data_path, formatted_local_fname)
 
         # perform download
-        try:
-            print('Attempting to download file for '+date.strftime('%x'))
-            sys.stdout.flush()
-            remote_path = '/'.join((remote_url, formatted_remote_fname))
-            req = requests.get(remote_path)
-            if req.status_code != 404:
-                open(saved_local_fname, 'wb').write(req.content)
-                print('Finished.')
-            else:
-                print('File not available for ' + date.strftime('%x'))
-        except requests.exceptions.RequestException as exception:
-            print('File not available for ' + date.strftime('%x'))
+        if not multi_file_day:
+            try:
+                print('Attempting to download file for ' + date.strftime('%x'))
```

Comment:
This date output should be updated to the same format used by load. Write out month name, day, year. 
date.strftime('%d %B %Y')

Reply:
fixed